### PR TITLE
Added parameter for explicit context

### PR DESCRIPTION
**Pull request by cklein**
A parameter `context_stack` is added to every function of the call stack.
If not provided (default value is `None`), py_zipkin will use the thread local storage
and work as expected.

### CODE REVIEW

File: py_zipkin/zipkin.py [hunk 1]
```diff
@@ -163,6 +164,10 @@ def __init__(
         :param host: Contains the ipv4 value of the host. The ipv4 value isn't
             automatically determined in a docker environment
         :type host: string
+        # CK45
```
Comment (bplotnick) [inline py_zipkin/zipkin.py]: What's this comment?

Comment (cklein) [reply py_zipkin/zipkin.py]: It was kind of a bookmark for me, forgot to remove it.

**Comment by cklein** [comment]
Hello @bplotnick,
sorry for the delay, I should have answered yesterday, since py_zipkin (as well as distributed_context) kind of blocks the whole project.
I changed my code according to how I understood your comments.
If there are further issues, I suggest that we can address them on IRC (or similar), to reduce the RTT.

**Comment by bplotnick** [comment]
This looks good to me. For future reference, try not to rebase on GH since it messes up the interdiff, and makes it hard to see what has changed 😐

**Pull request merged**